spec/rails_helper.rb

```diff
@@ -38,7 +38,7 @@
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
-
+  config.include Capybara::DSL
```

Comment:
is it worth having this in here if it's not necessary?

Reply:
It doesn’t hurt, and it might be necessary for me to run it on my machine.